[quant] Subpackage import in nn.quantized

z-a-f opened the pull request:
Some of the subpackages were not included in the 'torch.nn.quantized'.
That would cause some specific cases fail.
For example, `from torch.nn.quantized import dynamic` would work,
but `import torch; torch.nn.quantized.dynamic` would fail.

Fixes #ISSUE_NUMBER

z-a-f commented:
@pytorchbot rebase

z-a-f commented:
@pytorchbot rebase

andrewor14 commented on torch/nn/qat/__init__.py:
```diff
@@ -4,4 +4,15 @@
 This package is in the process of being deprecated.
 Please, use `torch.ao.nn.qat.dynamic` instead.
 """
+from . import dynamic  # noqa: F403
```
Do we need to do the same thing in `torch/nn/qat/dynamic/__init__.py`? Does this work right now: `import torch; torch.nn.qat.dynamic.Linear`?

andrewor14 replied:
Same for `torch/nn/quantized/dynamic/__init__.py`

z-a-f replied:
We don't need to do it for the `dynamic/__init__.py`, but that raises a point on using the `torch.nn.qat.dynamic.Linear` -- I will double check

z-a-f replied:
You were right, the `qat` was not visible to the users. Fixed

andrewor14 commented on torch/nn/quantized/__init__.py:
```diff
@@ -1,2 +1,40 @@
+    'ReLU6',
+    'Sigmoid',
+    'Softmax',
+    'Dropout',
```
nit: out of order alphabetically? (Looks like the rest is sorted)

Commit:
[quant] Subpackage import

Some of the subpackages were not included in the 'torch.nn.quantized'.
That would cause some specific cases fail.
For example, `from torch.nn.quantized import dynamic` would work,
but `import torch; torch.nn.quantized.dynamic` would fail.

z-a-f commented:
@pytorchbot merge -g

z-a-f commented:
@pytorchbot rebase

z-a-f commented:
@pytorchbot merge -g
